README.md

```diff
@@ -72,21 +72,41 @@ jobs:
     # the ${{ steps.buildnumber.outputs.build_number }} output.
 ```
 
+## Generating multiple independent build numbers
```

Comment:
This should be moved below the "Settings the initial build number" section.

Reply:
Moved